pkg/cmd/cmd.go

```diff
@@ -72,8 +72,24 @@ func NewDefaultGardenctlCommand() *cobra.Command {
 // NewGardenctlCommand creates the `gardenctl` command
 func NewGardenctlCommand(f *util.FactoryImpl, ioStreams util.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:          "gardenctl",
-		Short:        "gardenctl is a utility to interact with Gardener installations",
+		Use:   "gardenctl",
+		Short: "gardenctl is a utility to interact with Gardener installations",
+		Long: `Gardenctl is a utility to interact with Gardener installations.
+
+The state of gardenctl is bound to a shell session and is not shared across windows, tabs or panes.
+A shell session is defined by the environment variable GCTL_SESSION_ID. If this is not defined,
+the value of the TERM_SESSION_ID environment variable is used instead. If both are not defined,
+this leads to an error and gardenctl cannot be executed. The target.yaml and temporary
+kubeconfig.*.yaml files are store in the following directory ${TMPDIR}/garden/${GCTL_SESSION_ID}.
+
+You can make sure that GCTL_SESSION_ID or TERM_SESSION_ID is always present by adding
+the following code to your terminal profile ~/.profile, ~/.bashrc or comparable file.
+  bash and zsh: [ -n "$TERM_SESSION_ID" ] || export TERM_SESSION_ID="$(uuidgen)"
+  fish:         [ -n "$TERM_SESSION_ID" ] || set -gx TERM_SESSION_ID "$(uuidgen)"
+  powershell:   $Env:TERM_SESSION_ID ??= [guid]::NewGuid().ToString()
```

Comment:
just thought about it again, shouldn't we rather set the `GCTL_SESSION_ID` if neither `GCTL_SESSION_ID` nor `TERM_SESSION_ID` is set, instead of setting `TERM_SESSION_ID`?

Reply:
I agree. And maybe in fish shell it should be
```
set -gx TERM_SESSION_ID "(uuidgen)"
```

Reply:
✔️